### PR TITLE
added date functin for automatic date updates

### PR DESCRIPTION
**Pull request by WilloS88**

### CODE REVIEW

**Comment by SedlarDavid** [comment]
V souboru public/index.html je v headu stále rok 2023

**Review by SedlarDavid — CHANGES_REQUESTED** [review]
viz předchozí comment

File: index.html [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+      const GetYear = () => {
+        const actualYear = new Date().getFullYear();
+        document.title = `Majáles ${actualYear}`;
+      };
+    </script>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Majáles 2023</title>
-    <meta name="description" content="">
+    <title></title>
```
Comment (tomesfilip) [inline index.html:12]: missing title

File: index.html [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+    <meta name="description" content="" />
 
-    <meta property="og:locale" content="cs_CZ">
-    <meta property="og:type" content="website">
-    <meta property="og:title" content="Majáles 2023">
-    <meta property="og:description" content="">
+    <meta property="og:locale" content="cs_CZ" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Majáles 2023" />
```
Comment (tomesfilip) [inline index.html:17]: still 2023

**Review by tomesfilip — CHANGES_REQUESTED** [review]
okay, in public files (like index.html) we need to use hardcoded value. maybe for the sake of simplicity, let's use the hardcoded 2024 value everywhere. This will be changed in the new repo.

**Review by tomesfilip — APPROVED** [review]

**Pull request merged**